run_pretrained_openfold.py

```diff
@@ -334,54 +416,64 @@ def main(args):
                     )
                     cur_tracing_interval = rounded_seqlen
 
-            out = run_model(model, processed_feature_dict, tag, args.output_dir)
-
-            # Toss out the recycling dimensions --- we don't need them anymore
-            processed_feature_dict = tensor_tree_map(
-                lambda x: np.array(x[..., -1].cpu()),
-                processed_feature_dict
-            )
-            out = tensor_tree_map(lambda x: np.array(x.cpu()), out)
-
-            unrelaxed_protein = prep_output(
-                out,
-                processed_feature_dict,
-                feature_dict,
-                feature_processor,
-                args.config_preset,
-                args.multimer_ri_gap,
-                args.subtract_plddt
-            )
-
-            unrelaxed_file_suffix = "_unrelaxed.pdb"
-            if args.cif_output:
-                unrelaxed_file_suffix = "_unrelaxed.cif"
-            unrelaxed_output_path = os.path.join(
-                output_directory, f'{output_name}{unrelaxed_file_suffix}'
-            )
+            for seed in random_seeds:
+                np.random.seed(seed)
+                torch.manual_seed(seed + 1) # This is what the original code does
+
+                try:
+                    out = run_model(model, processed_feature_dict, tag, args.output_dir)
```

Comment:
I want to make sure that switching seeds right before calling inference on the model is equivalent to switching seeds earlier. One thing that comes to mind is if there is some stochasticity when `load_models_from_command_line` gets called.